validator/sawtooth_validator/journal/publisher.py

```diff
@@ -463,21 +494,22 @@ def on_check_publish_block(self, force=False):
                         self._identity_signing_key,
                         pending_batches)
                     self._candidate_block = None
+                    # In every case we get a pending batch list back.
+                    # Update the _pending_batches to reflect what we learned.
+                    last_batch_index = self._pending_batches.index(last_batch)
+                    additional_batches =\
+                        self._pending_batches[last_batch_index + 1:]
+                    # the pending batch list is now the list we have returned
+                    # from the candidate block plus all batche that were not
```

Comment:
batche -> batches

Also, I'm not sure this comment is necessary at all if you rename `additional_batches` to `unsent_batches` or something similar.